lib/__tests__/standalone-deprecations.test.mjs

```diff
@@ -1,26 +1,20 @@
-import { createRequire } from 'node:module';
 import readJSONFile from '../testUtils/readJSONFile.mjs';
 import standalone from '../standalone.mjs';
 
 import { jest } from '@jest/globals';
 
-jest.mock('../rules/block-no-empty/index.cjs');
-
-const require = createRequire(import.meta.url);
-const blockNoEmpty = require('../rules/block-no-empty/index.cjs');
+jest.unstable_mockModule('../rules/block-no-empty/index.mjs', () => ({
```

Comment:
[note] For `jest.unstable_mockModule`, see https://jestjs.io/docs/ecmascript-modules#module-mocking-in-esm